### PR TITLE
Bug 1541252 - review notes

### PR DESCRIPTION
**Pull request by jotes**
TODO:
* [x] - Remove unecessary hashes
* [x] - Use absolute imports instead of relative ones.

### CODE REVIEW

**Comment by jotes** [comment]
Hey,
This patch is huge, but it introduces 2 simple changes:
* add a `absolute_import` statement in almost every python file (skipped migrations and files without any imports)
* remove unecessary hashes in requirements.txt

@Pike do you have a moment to look at this?

File: pontoon/allauth_urls.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+from __future__ import absolute_import
```
Comment (Pike) [inline pontoon/allauth_urls.py]: ⬇️

File: pontoon/base/management/commands/updatefxaprovider.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+from __future__ import absolute_import
```
Comment (Pike) [inline pontoon/base/management/commands/updatefxaprovider.py]: ⬇️

File: pontoon/administration/tests/test_views.py [hunk 3]
```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
```
Comment (Pike) [inline pontoon/administration/tests/test_views.py]: extra newline

File: pontoon/base/tests/commands/test_user.py [hunk 4]
```diff
@@ -1,3 +1,5 @@
+from __future__ import absolute_import
+
```
Comment (Pike) [inline pontoon/base/tests/commands/test_user.py]: extra newline.

File: pontoon/base/tests/forms/test_entity.py [hunk 5]
```diff
@@ -1,3 +1,5 @@
+from __future__ import absolute_import
+
```
Comment (Pike) [inline pontoon/base/tests/forms/test_entity.py]: extra newline

File: pontoon/base/tests/managers/test_entity.py [hunk 6]
```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
```
Comment (Pike) [inline pontoon/base/tests/managers/test_entity.py]: extra newline

File: pontoon/base/tests/test_helpers.py [hunk 7]
```diff
@@ -1,4 +1,6 @@
 # coding: utf-8
+from __future__ import absolute_import
+
```
Comment (Pike) [inline pontoon/base/tests/test_helpers.py]: extra newline

File: pontoon/batch/forms.py [hunk 8]
```diff
@@ -1,3 +1,5 @@
+from __future__ import absolute_import
+
```
Comment (Pike) [inline pontoon/batch/forms.py]: extra newline

File: pontoon/contributors/apps.py [hunk 9]
```diff
@@ -1,3 +1,5 @@
+from __future__ import absolute_import
+
```
Comment (Pike) [inline pontoon/contributors/apps.py]: extra newline

File: pontoon/homepage/admin.py [hunk 10]
```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
```
Comment (Pike) [inline pontoon/homepage/admin.py]: extra newline

File: pontoon/homepage/apps.py [hunk 11]
```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
```
Comment (Pike) [inline pontoon/homepage/apps.py]: extra newline

**Comment by jotes** [comment]
@Pike thanks, I'll review everything again (I missed the imports above docstrings) and ping you when it's ready.

**Comment by jotes** [comment]
@Pike Hey,
I know that this patch is huge, so I'll try to make a quick summary of latest fixes:
* remove all extra newlines (there should be only 1 newline between `__future__` imports and the rest of imports).
* absolute imports are below module-level docstrings
* reduce the number of imports from `__future__` to a single import statement per file.

**Pull request merged**